upup/models/cloudup/_aws/topologies/_topology_private/network.yaml

```diff
@@ -190,6 +190,13 @@ loadBalancer/api.{{ ClusterName }}:
   {{ end }}
   listeners:
     443: { instancePort: 443 }
+  scheme: internal
```

Comment:
Changing the default scheme is a bigger change, and probably more controversial.  Can we split it out into another PR?